src/ORM/DataObject.php

```diff
@@ -2447,6 +2448,19 @@ public function getCMSActions()
         return $actions;
     }
 
+    public function getCMSValidators(): ValidatorList
+    {
+        $validatorList = new ValidatorList();
+
+        // Support for the old method during the deprecation period
+        if ($this->hasMethod('getCMSValidator')) {
+            $validatorList->addValidator($this->getCMSValidator());
```

Comment:
I assume this is where we’ll add the deprecation notice?

Reply:
Good question. I'm not really sure of the best practice there, but I'd be happy to do whatever makes sense.

Reply:
Yup, in this condition, throw `Deprecation::notice('4.6', 'getCMSValidator() is removed in 5.0 in favour of....')`

Just remember to update the 4.x version if this doesn't get merged before 4.6. :)

Reply:
Thanks, @unclecheese ! I've updated this now.